File: WebApplication1/QueryHandler/Query/GameQuery.cs
using MediatR;
using WebApplication1.DTO.Response;

namespace WebApplication1.QueryHandler.Query
{
    public class GameQuery:IRequest<List<GameResponse>>
    {
        public string? title { get; set; }
        public string? genreName { get; set; }

        public string? sortByField { get; set; }
        public bool IsDescending { get; set; }
    }
}

File: angular-movieFrontend/src/ClientPages/MainPages/game-web/GameQuery.ts
export interface GameQuery {
    TitleSearch?: string | null;
    MinRating?: number | null;
    Platform?: string | null;
    Developer?: string | null;
    ReleaseDate?: number | null;
    genreName?: string | null;
    
    SortByField?: string | null;
    IsDescending?: boolean | null;
}